smithy-diff/src/main/java/software/amazon/smithy/diff/evaluators/AddedEntityBinding.java

```diff
@@ -37,6 +38,8 @@
 public final class AddedEntityBinding extends AbstractDiffEvaluator {
     private static final String ADDED_RESOURCE = "AddedResourceBinding";
     private static final String ADDED_OPERATION = "AddedOperationBinding";
+    private static final String TO_RESOURCE = ".ToResourceAdded.";
```

Comment:
Same thing here -- we know it wa added by the event ID, no need to repeat that here. I suggest ".Resource" and ".Service"